package.json

```diff
@@ -2,22 +2,20 @@
   "name": "spectacle",
   "version": "5.5.0",
   "description": "ReactJS Powered Presentation Framework",
+  "source": "src/components/index.js",
   "main": "lib/index.js",
   "module": "es/index.js",
   "types": "index.d.ts",
   "scripts": {
-    "build-wds": "webpack-dev-server --config webpack.config.example.js",
-    "build-webpack-base": "webpack",
-    "build-webpack-example": "builder run build-webpack-base -- --config webpack.config.example.js",
-    "build-webpack-lib-dev": "builder run build-webpack-base -- --config webpack.config.dev.js",
-    "build-webpack-lib": "builder run build-webpack-base",
-    "build": "builder concurrent --buffer build-webpack-lib build-webpack-lib-dev build-webpack-example",
     "lint": "eslint --ignore-path .gitignore .",
     "lint-fix": "yarn lint --fix",
     "prettier": "prettier \"**/*.{js,json,ts,css,md}\"",
-    "prettier-fix": "yarn prettier --write",
+    "prettier-fix": "npm run prettier -- --write",
+    "rollup": "rollup -c config/rollup.config.js",
+    "dev-server": "yarn rollup --watch --open --port=3000",
+    "start": "NODE_ENV=development yarn dev-server",
+    "build": "NODE_ENV=production yarn rollup",
```

Comment:
Let's make it cross-platform compatible with `builder` execution:

```js
"start": "builder run --env \"{\\\"NODE_ENV\\\":\\\"development\\\"}\" dev-server",
"build": "builder run --env \"{\\\"NODE_ENV\\\":\\\"production\\\"}\" rollup",
```

Reply:
Quick suggestion, we could use cross-env here which is a little easier to read and more commonly used?

Reply:
For concurrent/parallel work, `builder concurrent` (run different `package.json:scripts` tasks) or `builder env` (run the same `package.json:scripts` task with different environment variables) is more flexible and powerful than `concurrently` and is also part of `builder`.

If we're additionally doing that, then I'd vote we stick with `builder` to just let one tool do everything. If not, then sure, go ahead with `cross-env`

On that note now, are we producing both minified and non-minified ("development") builds in `dist/` for the library? If not, we should consider doing that. (I mention it because what `builder` was here in the first place for last time...)